views/features.erb

```diff
@@ -146,8 +146,8 @@
             <div class="table-content">
               <h3>5 días por semana</h3>
               <h3>8hs</h3>
-              <h1>$2100</h1>
-              <sup>mensual + imp</sup>
+              <h1>$2600</h1>
+              <sup>mensual</sup>
```

Comment:
What about moving this to a helper method ?

Reply:
Good call!! Haven't thought about it!!